packages/clock/src/timers/index.ts

```diff
@@ -0,0 +1,27 @@
+import { callNextSecond } from './utils';
+
+/**
+ *
+ */
+export function setClockInterval(cb: () => void, ms: number) {
+  let timeout: number | undefined;
+
+  function repeat() {
```

Comment:
IDEA: every Nth repeat call attempt to "sync" to nearest second in case drift starts happening.

Reply:
This is now included, although I decided to do it every tick vs. after N ticks since it is so simplistic.